### ISSUE TITLE
SQLAlchemy 1.4.2 support

### ISSUE THREAD

**Issue by ysomad** [fastapi-users/fastapi-users#564]
## Describe the bug

Installing fastapi-users[sqlalchemy] and getting version solving error

## To Reproduce

- poetry add SQLAlchemy && poetry add fastapi-users[sqlalchemy]

## Expected behavior

install fastapi-users[sqlalchemy] dependency using sqlalchemy 1.4.2 and poetry

## Configuration
 - Python version : 3.9.2
 - FastAPI version : 0.63.0
 - FastAPI Users version : 5.1.2
 - SQLAlchemy version: 1.4.2

**Comment by ysomad** [comment]
> ## Describe the bug
> Installing fastapi-users[sqlalchemy] and getting version solving error
> 
> ## To Reproduce
> Steps to reproduce the behavior:
> 
> * poetry add SQLAlchemy && poetry add fastapi-users[sqlalchemy]
> 
> ## Expected behavior
> install fastapi-users[sqlalchemy] dependency using sqlalchemy 1.4.2 and poetry
> 
> ## Configuration
> * Python version : 3.9.2
> * FastAPI version : 0.63.0
> * FastAPI Users version : 5.1.2
> * SQLAlchemy version: 1.4.2

```
Because no versions of fastapi-users match >5.1.2,<6.0.0
   and fastapi-users (5.1.2) depends on sqlalchemy (>=1.3.13,<1.4), fastapi-users (>=5.1.2,<6.0.0) requires sqlalchemy (>=1.3.13,<1.4).
  So, because yopta depends on both SQLAlchemy (^1.4.2) and fastapi-users (^5.1.2), version solving failed.
```

**Comment by ysomad** [comment]
SolverProblemError raised
`Because no versions of fastapi-users match >5.1.2,<6.0.0
   and fastapi-users (5.1.2) depends on sqlalchemy (>=1.3.13,<1.4), fastapi-users (>=5.1.2,<6.0.0) requires sqlalchemy (>=1.3.13,<1.4).
  So, because yopta depends on both SQLAlchemy (^1.4.2) and fastapi-users (^5.1.2), version solving failed.`

**Comment by frankie567** [comment]
Currently, we are blocked by `encode/databases` which doesn't support SQLAlchemy 1.4 versions. See https://github.com/frankie567/fastapi-users/discussions/554#discussioncomment-503593

Meanwhile, please pin SQLAlchemy to 1.3.x.

**Comment by ysomad** [comment]
got it, thank you!

**Comment by badarsebard** [comment]
Looks like the `encode/databases` project fixed this in the 0.5.0 release a few days ago.

**Comment by frankie567** [comment]
Indeed! Actually, I'm currently splitting all the database adapters [into separate packages](https://github.com/fastapi-users) and I've already updated the one for SQLAlchemy/databases. I'll release it soon!

**Comment by frankie567** [comment]
Should be fixed in [`v7.0.0`](https://github.com/fastapi-users/fastapi-users/releases/tag/v7.0.0) with updated [SQLAlchemy/Databases adapter](https://github.com/fastapi-users/fastapi-users-db-sqlalchemy).
